### PR TITLE
Small fix + merge master

### PR DESCRIPTION
**Pull request by Mariocj89**
Bring changes from master
Fix translation issue

### CODE REVIEW

File: server/forms/link_sets_form.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def __init__(self, *args, **kwargs):
         self.helper.field_class = 'col-xs-9'
         self.helper.layout = Layout(
             Row(
-                HTML("<div class='alert alert-info' role='alert'>{0}</div>".format(_("Separate items by commas. e.g: Maria, David S, Leo, ..."))),
+                HTML(_("<div class='alert alert-info' role='alert'>Separate items by commas. e.g: Maria, David S, Leo, ...</div>")),
```
Comment (dnaranjo89) [inline server/forms/link_sets_form.py]: Why this change? The whole component is being translated

Comment (Mariocj89) [inline server/forms/link_sets_form.py]: Check it. It was not appearing otherwise
